Source/Globals.cpp

```diff
@@ -59,7 +61,23 @@ namespace android::global
     void Initialize(JavaVM* javaVM, jobject context)
```

Comment:
@lmaskati is there any risk of two threads trying to initialize at the same time?

Reply:
I think this is more of a question for @bghgary, so tagging him here since he wrote that code.

Reply:
This function should only be called once from the main Java thread of the application. But this isn't currently documented, so we probably should make that explicitly written somewhere.